src/components/SendTransaction.vue

```diff
@@ -213,6 +224,24 @@ function calculateCurrencyValue(value) {
   <div v-if="loader.show" class="flex justify-center items-center flex-1 p-4">
     <AppLoader :message="loader.message" />
   </div>
+  <div v-else-if="shrinkMode" class="flex justify-between p-3">
```

Comment:
What is shrinkMode?

Reply:
shrink mode is to show pending tasks in the activities tab